File: 프로그래머스/0/120869. 외계어 사전/외계어 사전.py
def solution(spell, dic):
    answer = 0
    
    for i in dic :
        dict = {}
        for k in spell :
            dict[k] = 0
        
        for j in spell :
            dict[j] = i.count(j)
        
        if 0 not in list(dict.values()) :
            return 1
    
    return 2

File: 프로그래머스/0/120869. 외계어 사전/외계어 사전.java
class Solution {
    public int solution(String[] spell, String[] dic) {
        int answer = 0;
        
        for (String d : dic) {
            boolean isContain = true;
            for (String s : spell) {
                if (!d.contains(s)) {
                    isContain = false;
                }
            }
            if (isContain) {
                return 1;
            }
        }
        
        return 2;
    }
}